### PR TITLE
Add some stop locations enums

### PR DESCRIPTION
**Pull request by datanel**

### CODE REVIEW

**Comment by woshilapin** [comment]
Do we plan to do something similar for GTFS?

**Comment by datanel** [comment]
@woshilapin done https://github.com/CanalTP/transit_model/pull/331/commits/2f895602b9748e799a2d9be1ddf5ab5f3af38ff3

**Comment by woshilapin** [comment]
Yipee, I was just asking and then you just did it, wonderful!

File: src/ntfs/read.rs [hunk 1]
```diff
@@ -97,8 +98,16 @@ pub fn manage_stops(collections: &mut Collections, path: &path::Path) -> Result<
                 }
                 stop_points.push(StopPoint::from_with_type(stop, stop_type));
             }
-            1 => stop_areas.push(StopArea::from(stop)),
-            i => warn!("stop.location_type = {} not yet supported, skipping.", i),
+            StopLocationType::StopArea => stop_areas.push(StopArea::from(stop)),
+            StopLocationType::EntranceExit => {
```
Comment (woshilapin) [inline src/ntfs/read.rs]: I was wondering if something _like_ the following (the code below doesn't build) was possible but not really sure how and it's just a curiosity (I'm fine if we merge as it is).

```
stop_location_type => warn!(
  "stop.location_type = {} not yet supported, skipping.", 
  stop_location_type.serialize(?serializer?)
),
```

**Comment by woshilapin** [comment]
I merged the PR even if the build was broken because:
- the only broken build was `cargo fmt`
- the broken job was broken because of the cache problem
- I checked locally that `cargo fmt` was just fine

**Pull request merged**